ckanext/saml2auth/views/saml2auth.py

```diff
@@ -128,12 +130,34 @@ def acs():
     return resp
 
 
+def get_requested_authn_contexts():
+    requested_authn_contexts = config.get('ckanext.saml2auth.requested_authn_context', None)
+    if requested_authn_contexts is None:
+        return []
+
+    return requested_authn_contexts.strip().split()
```

Comment:
I forget how `split()` works, do you need a compact() to remove empty strings in the case of a double space between two reqs?

Reply:
```
a = "one two  three      four"
a.strip().split()
['one', 'two', 'three', 'four']
```
I think it's ok